test/src/api/endpoints/test-work.js

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable */
+
+process.env.NODE_ENV = 'test';
```

Comment:
I don't think you should need this. I don't think it's a good idea to leave it there either.
It should be set when using `npm test`